DQM/Integration/python/clients/ppsrandom_dqm_sourceclient-live_cfg.py

```diff
@@ -0,0 +1,97 @@
+import FWCore.ParameterSet.Config as cms
+
+import sys
+from Configuration.Eras.Era_Run3_cff import Run3
+process = cms.Process('CTPPSDQM', Run3)
+
+test = False
+unitTest = False
+
+if 'unitTest=True' in sys.argv:
+  unitTest=True
+
+# event source
+if unitTest:
+  process.load("DQM.Integration.config.unittestinputsource_cfi")
+  from DQM.Integration.config.unittestinputsource_cfi import options
+elif not test:
+  # for live online DQM in P5
+  process.load("DQM.Integration.config.inputsource_cfi")
+  from DQM.Integration.config.inputsource_cfi import options
+else:
+  # for testing in lxplus
+  process.load("DQM.Integration.config.fileinputsource_cfi")
+  from DQM.Integration.config.fileinputsource_cfi import options
+  process.source.fileNames = cms.untracked.vstring(
+    "/store/group/dpg_ctpps/comm_ctpps/PixelRandomTrigger2023/outputExpressPPSRandom.root"
+  )
+  process.source.inputCommands = cms.untracked.vstring(
+    'drop *',
+    'keep FEDRawDataCollection_*_*_*'
+  )
+  
+process.source.streamLabel = "streamExpressPPSRandom"
```

Comment:
From what I understood from the HLT report in the Daily Run Meetin (http://schwick.web.cern.ch/schwick/rctools/dailyReport/detailReport/display/2023-03-23) this should be updated to:
```suggestion
process.source.streamLabel = "streamDQMPPSRandom"
```

Reply:
Indeed! Thanks Francesco!